### ISSUE TITLE
Add fullscreen option to commandline 

### ISSUE THREAD

**Issue by iamthesenate1** [lxqt/lximage-qt#431]
##### Context

I think it would be useful to be able to just add a fullscreen option especially for tiling WMs where, if there are many windows opened, the requested image would be too small.

**Comment by tsujan** [comment]
I don't use tiling WMs but I agree that such a command-line option may be useful with them. So, I made a PR here: https://github.com/lxqt/lximage-qt/pull/432

**Comment by iamthesenate1** [comment]
It works perfectly. Thank you very much.

**Comment by tsujan** [comment]
Thanks for testing and also for the idea!

**Comment by tsujan** [comment]
@iamthesenate1

I forgot the new full-screen argument in the introspection data. As a result, new windows couldn't be opened after the first window. I added it in https://github.com/lxqt/lximage-qt/pull/434. For a fully functioning app, you need that branch.

**Comment by tsujan** [comment]
@iamthesenate1

If you try that branch, please tell me about the result of your tests, especially in regard to https://github.com/lxqt/lximage-qt/issues/433. I don't want the above-mentioned problem to remain in the git source for a long time.

@stefonarch 
Would you please also test https://github.com/lxqt/lximage-qt/pull/434? It should fix https://github.com/lxqt/lximage-qt/issues/433.

**Comment by stefonarch** [comment]
Done, bug disappeared, but didn't test further.

**Comment by iamthesenate1** [comment]
Yes, it works now as intended. Didn't spot the multiple window problem as I usually open one window at a time, but it is indeed fixed, just like the thumbnail bar one.

Sorry for not being able to test this sooner.

**Comment by tsujan** [comment]
> just like the thumbnail bar one.

Yes, the thumbnail bar also had a bug in the fullscreen mode. It was hidden when the window became fullscreen, which is quite reasonable because the user expects the whole screen to be used by the image. But there was a problem when the thumbnail bar was explicitly shown in the fullscreen mode. Everything is fixed now.
